### DISCUSSION TITLE
IMAP & POP3 simultaneously

### DISCUSSION THREAD

**Discussion by wencywww** [docker-mailserver/docker-mailserver#2644] [Q&A] [answered]
Hello, is it possible POP3 / IMAP services to be used simultaneously or we need to choose one of them?
According to the info [here](https://hub.docker.com/r/mailserver/docker-mailserver/) :
...
[Dovecot](https://www.dovecot.org/) for SASL, IMAP (or POP3), with LDAP Auth, Sieve and [quotas](https://docker-mailserver.github.io/docker-mailserver/edge/config/user-management/accounts#notes)
...
Dovecot is supported for IMAP or POP3. I believe it should be read as and/or - right?

**Comment by casperklein** [comment, marked as answer]
You _can_ use both, but whether that's wise is another question 😉 

Keep in mind, if you are using POP3 and deleting mails from the server, those mails will obviously also not be available via IMAP anymore.
Some POP3 clients catch mails from the server and delete them afterwards or after a given time.

So to not experience any nasty surprises, I advice to do some testing first.

**Reply by wencywww** [reply]
:) Thanks, the testing is mandatory, of course :)
